ci/cloudwatch_lambda_test.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+shopt -s inherit_errexit
+
+src/dev set-up-ci-environment
```

Comment:
Do we need separate CI scripts to run the tests for each Lambda separately? I think we could run all the tests at once?

We just need to ensure all the tests pass

Reply:
I don't see the harm in having CI run the tests for both Lambdas at once